src/Components/Quiz/Quiz.js

```diff
@@ -5,18 +5,57 @@ import Question from '../Question/Question';
 const Quiz = ({ songData }) => {
   const { decade } = useParams();
 
-  const [count, setCount] = useState(0);
-  const [years, setYears] = useState(songData[decade]);
+  const [questionCount, setQuestionCount] = useState(0);
+  const [allSongs, setAllSongs] = useState(songData[decade]);
+  const [score, setScore] = useState({});
+  const [correctAnswers, setCorrectAnswers] = useState({});
+  const [playerAnswers, setPlayerAnswers] = useState({});
 
-  return (count <= 9) ? (
+  useEffect(() => {
+    getCorrectAnswers();
+  }, []);
+
+  const getCorrectAnswers = () => {
+    const allCorrectAnswers = allSongs.reduce((acc, item) => {
+      const year = Object.keys(item)[0];
+      const songId = item[year].song1.id;
```

Comment:
Is the .song1 supposed to be dynamic? Or are you deliberately wanting them all to have the same id?